volestipy/volestipy/volestipy.pyx

```diff
@@ -225,7 +345,7 @@ cdef class low_dim_HPolytope:
       return np.asarray(self._Aeq)
    @property
    def beq(self):
-      return np.asarray(self._beq)   
+      return np.asarray(self._beq)
    @property
    def dimensions(self):
-      return self._A.shape[1]   
\ No newline at end of file
+      return self._A.shape[1]
```

Comment:
please add an empty line at the end of all files that contain code in folder `volestipy`

Reply:
done.

Reply:
these are four lines, one is enough ;)

Reply:
hmm, you are back to the empty line issue now ;)

Reply:
There is something strange with my editor I think..
If I am not mistaken, last line in the ```volestipy.pyx``` file is number 571 and it is empty..
Is it not?